sslcertificates/agents/plugins/sslcertificates

```diff
@@ -51,7 +51,7 @@ get_cert_info() {
             cert_algosign=$($OPENSSL x509 -inform $inform -noout -text -in "$certfile" | awk '/Signature Algorithm: / { print $3; exit;}' )
             cert_issuer_hash=$($OPENSSL x509 -inform $inform -noout -issuer_hash -in "$certfile" )
 
-            echo "{\"file\": \"$certfile\", \"starts\": $cert_startdate_epoch, \"expires\": $cert_enddate_epoch, \"algosign\": \"$cert_algosign\", \"issuer_hash\": \"$cert_issuer_hash\", \"subj\": \"$cert_subject\"}"
+            echo "{\"file\": \"$certfile\", \"starts\": $cert_startdate_epoch, \"expires\": $cert_enddate_epoch, \"algosign\": \"$cert_algosign\", \"issuer_hash\": \"$cert_issuer_hash\", \"subj\": \"$cert_subject\", \"template\": null}"
```

Comment:
This can be removed as you already use "if data.get('template')" in the check plugin.

No need to add not needed data.